index.html

```diff
@@ -1,199 +1,315 @@
+                        <h3>Multi-Post Stories Gain + Glory</h3>
+                        <div class="programing-languages">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>Javascript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <button class="green-button">See project</button>
+                    </div>
+    
             </section>
-            <article class="card-holder">
-                <section>
-                    <div class="card">
+        </article>
+        <article class="card-holder">
+            <section>
+                <div class="card">
                     <div class="empty-box">
-        
+    
                     </div>
                     <div class="multi-post-stories">
                         <h3>Multi-Post Stories Gain + Glory</h3>
                         <div class="programing-languages">
-                        <ul>
-                            <li>Ruby on Rails</li>
-                            <li>CSS</li>
-                            <li>Javascript</li>
-                            <li>HTML</li>
-                    </ul>
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>Javascript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <button class="green-button">See project</button>
                     </div>
-                    <button class="green-button">See project</button>
+    
+            </section>
+        </article>
+        <article class="card-holder">
+            <section>
+                <div class="card">
+                    <div class="empty-box">
+    
                     </div>
-                    
-                </section>
-                <article class="card-holder">
-                    <section>
-                        <div class="card">
-                        <div class="empty-box">
-            
+                    <div class="multi-post-stories">
+                        <h3>Multi-Post Stories Gain + Glory</h3>
+                        <div class="programing-languages">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>Javascript</li>
+                                <li>HTML</li>
+                            </ul>
                         </div>
-                        <div class="multi-post-stories">
-                            <h3>Multi-Post Stories Gain + Glory</h3>
-                            <div class="programing-languages">
+                        <button class="green-button">See project</button>
```

Comment:
- [ ] According to your previouse reviewer buttons are missing styles for the active and hover states, as per the design kindly fix it to male your website look professional and user friendly.
- [ ] 
![Screenshot 2023-06-09 032853](https://github.com/Young-Gaius/portfolio-setup/assets/92458236/60791ab2-126d-4e87-a4f5-eecdfb4f9f2a)